### PR TITLE
Add hacktoberfest to website homepage (updated)

### PR DESCRIPTION
**Pull request by tmott13**
Added Hacktoberfest details to website homepage in the src folder.
Link to issue: https://github.com/get-alex/alex/issues/199

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -176,6 +176,9 @@ <h2 id="contributing">Contributing</h2>
         <h1>Made with love.</h1>
         <p>By <a href="https://twitter.com/wooorm">@wooorm</a>.</p>
         <p><small>&hellip;and a special thanks to <a href="https://twitter.com/sindresorhus">@sindresorhus</a> for inspiring me to do this.</small></p>
+        <br/>
```
Comment (wooorm) [inline src/index.html]: Hmm, what’s the `<br`> doing here? Is that needed?

File: src/index.html [hunk 2]
```diff
@@ -176,6 +176,9 @@ <h2 id="contributing">Contributing</h2>
...
+        <p>Contribute to Alex for the GitHub Social Impact Collection Challenge during the month of October for Hacktoberfest to receive a limited edition badge on your GitHub Community Forum profile.  For more information, please click the image below.</p>
```
Comment (wooorm) [inline src/index.html]: Maybe a link here? https://github.com/collections/social-impact

Comment (wooorm) [reply src/index.html]: Maybe both?!

Hey, we could also add this more at the start of the page? This is very low!

Comment (tmott13) [reply src/index.html]: how does this look? 
hacktoberfest_screenshot2

Comment (tmott13) [reply src/index.html]: or put the text under the image? 
hacktoberfest_screenshot3

Comment (wooorm) [reply src/index.html]: I like the last one, image first, text second.

* If you still need some spacing, maybe a heading makes more sense semantically than a `<br>` element. Or some CSS?
* For the link in the text, could you make it a bit more descriptive? `click here` is not so good for [accessibility or SEO](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/)
* And could you remove `target="_blank"` from the link? It’s fine if the link loads in the original tab/window, I think, but maybe you had a reason for it!

**Comment by tmott13** [comment]
I’ll make those changes for you. I’ll paste in the link instead of “click here”.  Opening up links in a new tab/window is just my personal preference but I will fix it for you. Any additional text you would like me to add?

**Comment by tmott13** [comment]
Or I can make “GitHub Social Impact Collection Challenge” text in the paragraph the link. That way would be much better I think.

**Comment by tmott13** [comment]
How does this look? I removed the `target="_blank"` in the link. No breaks are being used.

hacktoberfest_screenshot4

**Comment by wooorm** [comment]
Perfect!

**Comment by wooorm** [comment]
Oh it looks like you haven’t pushed the code yet, could you do that, and I’ll merge! 👍

**Comment by tmott13** [comment]
I added the updated index.html file

**Comment by wooorm** [comment]
@tmott13 Awesome Tanya, thank you! Congratulations on your first merged PR here! 🎉🎉🎉

**Comment by tmott13** [comment]
Thank you so much! 🎉🎉🎉

**Pull request merged**